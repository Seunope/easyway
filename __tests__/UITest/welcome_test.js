import React from 'react';
import {create} from 'react-test-renderer';
import Welcome from '../../src/screens/welcome/welcome';

const tree = create(<Welcome />);

test('snapshot', () => {
  expect(tree).toMatchSnapshot();
});
