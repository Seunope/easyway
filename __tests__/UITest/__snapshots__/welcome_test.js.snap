// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "flex": 1,
      "height": 1334,
    }
  }
>
  <View
    style={
      Array [
        Object {},
        Object {
          "alignItems": "center",
          "backgroundColor": "#FFFFFF",
          "flex": 5,
          "justifyContent": "center",
          "padding": 20,
        },
      ]
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#000",
            "fontFamily": "System",
            "fontSize": 16,
          },
          Object {
            "color": "#000000",
            "fontFamily": "Muli-ExtraBold",
            "fontSize": 62.857142857142854,
            "marginBottom": 10,
          },
        ]
      }
      uppercase={false}
    >
      EasyWay
    </Text>
    <Text
      style={
        Object {
          "color": "#000",
          "fontFamily": "System",
          "fontSize": 16,
        }
      }
      uppercase={false}
    >
      Going some place? Find and Go!
    </Text>
  </View>
  <View
    style={
      Array [
        Object {},
        Object {
          "alignItems": "center",
          "backgroundColor": "#FFFFFF",
          "flex": 2,
          "justifyContent": "flex-start",
        },
      ]
    }
  >
    <View
      style={
        Array [
          Object {
            "flexDirection": "row",
            "height": 60,
            "justifyContent": "space-between",
            "opacity": 1,
            "width": 60,
          },
          undefined,
        ]
      }
    >
      <View
        style={
          Object {
            "backgroundColor": "#FF6600",
            "height": 60,
            "transform": Array [
              Object {
                "scaleY": 0.5390648897816493,
              },
            ],
            "width": 9,
          }
        }
      />
      <View
        style={
          Object {
            "backgroundColor": "#FF6600",
            "height": 60,
            "transform": Array [
              Object {
                "scaleY": 0.9225028413716081,
              },
            ],
            "width": 9,
          }
        }
      />
      <View
        style={
          Object {
            "backgroundColor": "#FF6600",
            "height": 60,
            "transform": Array [
              Object {
                "scaleY": 0.9663215083723175,
              },
            ],
            "width": 9,
          }
        }
      />
      <View
        style={
          Object {
            "backgroundColor": "#FF6600",
            "height": 60,
            "transform": Array [
              Object {
                "scaleY": 0.6152958379564872,
              },
            ],
            "width": 9,
          }
        }
      />
      <View
        style={
          Object {
            "backgroundColor": "#FF6600",
            "height": 60,
            "transform": Array [
              Object {
                "scaleY": 0.4,
              },
            ],
            "width": 9,
          }
        }
      />
    </View>
  </View>
  <View
    style={
      Array [
        Object {},
        Object {
          "alignItems": "center",
          "backgroundColor": "#FFFFFF",
          "justifyContent": "center",
          "padding": 10,
        },
      ]
    }
  >
    <Text
      note={true}
      style={
        Object {
          "color": "#a7a7a7",
          "fontFamily": "System",
          "fontSize": 14,
        }
      }
      uppercase={false}
    >
      Powered By 
      <Text
        style={
          Array [
            Object {
              "color": "#000",
              "fontFamily": "System",
              "fontSize": 16,
            },
            Object {
              "color": "#FBBD08",
              "fontFamily": "Muli-Bold",
            },
          ]
        }
        uppercase={false}
      >
        Varens
      </Text>
    </Text>
  </View>
</View>
`;
